was AnimatedContentScope replaced correctely with AnimatedContentTransitionScope in the new Updates ?

i keep getting that the deprecated class AnimatedContentScope was not found and i already replaced it by AnimatedContentTransitionScope 
error ; 
E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.dzartxstudio.mybookofrhymes, PID: 12602
    java.lang.NoClassDefFoundError: Failed resolution of: Landroidx/compose/animation/AnimatedContentScope;
        at com.google.accompanist.navigation.animation.AnimatedNavHostKt$AnimatedNavHost$7$1.invoke(AnimatedNavHost.kt:198)
        at androidx.compose.animation.AnimatedContentKt.AnimatedContent(AnimatedContent.kt:759)
        at com.google.accompanist.navigation.animation.AnimatedNavHostKt.AnimatedNavHost(AnimatedNavHost.kt:196)
        at com.google.accompanist.navigation.animation.AnimatedNavHostKt.AnimatedNavHost(AnimatedNavHost.kt:89)
        at com.dzartxstudio.mybookofrhymes.navigation.NavGraphKt.NavGraph(NavGraph.kt:34)
        at com.dzartxstudio.mybookofrhymes.presentation.MainActivity$onCreate$1$2.invoke(MainActivity.kt:68)
        at com.dzartxstudio.mybookofrhymes.presentation.MainActivity$onCreate$1$2.invoke(MainActivity.kt:65)
        at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:116)
        at androidx.compose.runtime.internal.ComposableLambdaImpl$invoke$1.invoke(ComposableLambda.jvm.kt:127)
        at androidx.compose.runtime.internal.ComposableLambdaImpl$invoke$1.invoke(ComposableLambda.jvm.kt:127)
        at androidx.compose.runtime.RecomposeScopeImpl.compose(RecomposeScopeImpl.kt:162)
        at androidx.compose.runtime.ComposerImpl.recomposeToGroupEnd(Composer.kt:2443)
        at androidx.compose.runtime.ComposerImpl.skipCurrentGroup(Composer.kt:2711)
        at androidx.compose.runtime.ComposerImpl$doCompose$2$5.invoke(Composer.kt:3342)
        at androidx.compose.runtime.ComposerImpl$doCompose$2$5.invoke(Composer.kt:3320)
        at androidx.compose.runtime.SnapshotStateKt__DerivedStateKt.observeDerivedStateRecalculations(DerivedState.kt:341)
        at androidx.compose.runtime.SnapshotStateKt.observeDerivedStateRecalculations(Unknown Source:1)
        at androidx.compose.runtime.ComposerImpl.doCompose(Composer.kt:3320)
        at androidx.compose.runtime.ComposerImpl.recompose$runtime_release(Composer.kt:3285)
        at androidx.compose.runtime.CompositionImpl.recompose(Composition.kt:772)
        at androidx.compose.runtime.Recomposer.performRecompose(Recomposer.kt:1047)
        at androidx.compose.runtime.Recomposer.access$performRecompose(Recomposer.kt:124)
        at androidx.compose.runtime.Recomposer$runRecomposeAndApplyChanges$2$2.invoke(Recomposer.kt:541)
        at androidx.compose.runtime.Recomposer$runRecomposeAndApplyChanges$2$2.invoke(Recomposer.kt:510)
        at androidx.compose.ui.platform.AndroidUiFrameClock$withFrameNanos$2$callback$1.doFrame(AndroidUiFrameClock.android.kt:34)
        at androidx.compose.ui.platform.AndroidUiDispatcher.performFrameDispatch(AndroidUiDispatcher.android.kt:109)
        at androidx.compose.ui.platform.AndroidUiDispatcher.access$performFrameDispatch(AndroidUiDispatcher.android.kt:41)
        at androidx.compose.ui.platform.AndroidUiDispatcher$dispatchCallback$1.doFrame(AndroidUiDispatcher.android.kt:69)
        at android.view.Choreographer$CallbackRecord.run(Choreographer.java:970)
        at android.view.Choreographer.doCallbacks(Choreographer.java:796)
        at android.view.Choreographer.doFrame(Choreographer.java:727)
        at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:957)
        at android.os.Handler.handleCallback(Handler.java:938)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:223)
        at android.app.ActivityThread.main(ActivityThread.java:7656)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
    	Suppressed: kotlinx.coroutines.DiagnosticCoroutineContextException: [androidx.compose.runtime.PausableMonotonicFrameClock@8dcee37, androidx.compose.ui.platform.MotionDurationScaleImpl@c305fa4, StandaloneCoroutine{Cancelling}@a94e10d, AndroidUiDispatcher@b03f8c2]
E/AndroidRuntime: Caused by: java.lang.ClassNotFoundException: Didn't find class "androidx.compose.animation.AnimatedContentScope" on path: DexPathList[[dex file "/data/data/com.dzartxstudio.mybookofrhymes/code_cache/.overlay/base.apk/classes11.dex", dex file "/data/data/com.dzartxstudio.mybookofrhymes/code_cache/.overlay/base.apk/classes8.dex", zip file "/data/app/~~IPqPGsvT8t3sgk4X3Aokdw==/com.dzartxstudio.mybookofrhymes-obV70S1XTUo9vYrpMuBZVA==/base.apk"],nativeLibraryDirectories=[/data/app/~~IPqPGsvT8t3sgk4X3Aokdw==/com.dzartxstudio.mybookofrhymes-obV70S1XTUo9vYrpMuBZVA==/lib/x86, /system/lib, /system_ext/lib]]
        at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:207)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
        	... 39 more

i got the same problem

java.lang.NoClassDefFoundError: Failed resolution of: Landroidx/compose/animation/AnimatedContentScope;

Caused by: java.lang.ClassNotFoundException: Didn't find class "androidx.compose.animation.AnimatedContentScope" on path: DexPathList...

--- a/README.md
+++ b/README.md
@@ -24,7 +24,10 @@ Each [release](https://github.com/google/accompanist/releases) outlines what ver
   <td>Compose UI 1.3 (1.3.x)</td><td><img alt="Maven Central" src="https://img.shields.io/maven-central/v/com.google.accompanist/accompanist-permissions?versionPrefix=0.28"></td>
  </tr>
  <tr>
-  <td>Compose UI 1.4 (1.4.x)</td><td><img alt="Maven Central" src="https://img.shields.io/maven-central/v/com.google.accompanist/accompanist-permissions"></td>
+  <td>Compose UI 1.4 (1.4.x)</td><td><img alt="Maven Central" src="https://img.shields.io/maven-central/v/com.google.accompanist/accompanist-permissions?versionPrefix=0.30"></td>
+ </tr>
+ <tr>
+  <td>Compose UI 1.5 (1.5.x)</td><td><img alt="Maven Central" src="https://img.shields.io/maven-central/v/com.google.accompanist/accompanist-permissions"></td>
  </tr>
 </table>
 

--- a/gradle.properties
+++ b/gradle.properties
@@ -33,7 +33,7 @@ systemProp.org.gradle.internal.http.socketTimeout=120000
 
 GROUP=com.google.accompanist
 # !! No longer need to update this manually when using a Compose SNAPSHOT
-VERSION_NAME=0.30.2-SNAPSHOT
+VERSION_NAME=0.31.0-alpha
 
 POM_DESCRIPTION=Utilities for Jetpack Compose
 

--- a/gradle/libs.versions.toml
+++ b/gradle/libs.versions.toml
@@ -1,6 +1,6 @@
 [versions]
 
-compose = "1.4.0"
+compose = "1.5.0-alpha01"
 composeCompiler = "1.4.3"
 composeMaterial3 = "1.0.1"
 composesnapshot = "-" # a single character = no snapshot
@@ -16,7 +16,7 @@ okhttp = "3.12.13"
 coil = "1.3.2"
 
 androidxtest = "1.4.0"
-androidxnavigation = "2.5.3"
+androidxnavigation = "2.6.0-alpha08"
 androidxWindow = "1.0.0"
 
 [libraries]

--- a/navigation-animation/src/androidTest/java/com/google/accompanist/navigation/animation/AnimatedNavHostTest.kt
+++ b/navigation-animation/src/androidTest/java/com/google/accompanist/navigation/animation/AnimatedNavHostTest.kt
@@ -66,7 +66,7 @@ class AnimatedNavHostTest {
         composeTestRule.mainClock.autoAdvance = true
 
         composeTestRule.runOnIdle {
-            assertThat(firstEntry?.getLifecycle()?.currentState)
+            assertThat(firstEntry?.lifecycle?.currentState)
                 .isEqualTo(Lifecycle.State.RESUMED)
         }
 
@@ -76,17 +76,17 @@ class AnimatedNavHostTest {
             navController.navigate(second)
         }
 
-        assertThat(firstEntry?.getLifecycle()?.currentState)
+        assertThat(firstEntry?.lifecycle?.currentState)
             .isEqualTo(Lifecycle.State.CREATED)
-        assertThat(navController.currentBackStackEntry?.getLifecycle()?.currentState)
+        assertThat(navController.currentBackStackEntry?.lifecycle?.currentState)
             .isEqualTo(Lifecycle.State.STARTED)
 
         // advance half way between the crossfade
         composeTestRule.mainClock.advanceTimeBy(AnimationConstants.DefaultDurationMillis.toLong() / 2)
 
-        assertThat(firstEntry?.getLifecycle()?.currentState)
+        assertThat(firstEntry?.lifecycle?.currentState)
             .isEqualTo(Lifecycle.State.CREATED)
-        assertThat(navController.currentBackStackEntry?.getLifecycle()?.currentState)
+        assertThat(navController.currentBackStackEntry?.lifecycle?.currentState)
             .isEqualTo(Lifecycle.State.STARTED)
 
         composeTestRule.onNodeWithText(first).assertExists()
@@ -95,9 +95,9 @@ class AnimatedNavHostTest {
         composeTestRule.mainClock.autoAdvance = true
 
         composeTestRule.runOnIdle {
-            assertThat(firstEntry?.getLifecycle()?.currentState)
+            assertThat(firstEntry?.lifecycle?.currentState)
                 .isEqualTo(Lifecycle.State.CREATED)
-            assertThat(navController.currentBackStackEntry?.getLifecycle()?.currentState)
+            assertThat(navController.currentBackStackEntry?.lifecycle?.currentState)
                 .isEqualTo(Lifecycle.State.RESUMED)
         }
 
@@ -109,17 +109,17 @@ class AnimatedNavHostTest {
             navController.popBackStack()
         }
 
-        assertThat(navController.currentBackStackEntry?.getLifecycle()?.currentState)
+        assertThat(navController.currentBackStackEntry?.lifecycle?.currentState)
             .isEqualTo(Lifecycle.State.STARTED)
-        assertThat(secondEntry?.getLifecycle()?.currentState)
+        assertThat(secondEntry?.lifecycle?.currentState)
             .isEqualTo(Lifecycle.State.CREATED)
 
         // advance half way between the crossfade
         composeTestRule.mainClock.advanceTimeBy(AnimationConstants.DefaultDurationMillis.toLong() / 2)
 
-        assertThat(navController.currentBackStackEntry?.getLifecycle()?.currentState)
+        assertThat(navController.currentBackStackEntry?.lifecycle?.currentState)
             .isEqualTo(Lifecycle.State.STARTED)
-        assertThat(secondEntry?.getLifecycle()?.currentState)
+        assertThat(secondEntry?.lifecycle?.currentState)
             .isEqualTo(Lifecycle.State.CREATED)
 
         composeTestRule.onNodeWithText(first).assertExists()
@@ -128,9 +128,9 @@ class AnimatedNavHostTest {
         composeTestRule.mainClock.autoAdvance = true
 
         composeTestRule.runOnIdle {
-            assertThat(navController.currentBackStackEntry?.getLifecycle()?.currentState)
+            assertThat(navController.currentBackStackEntry?.lifecycle?.currentState)
                 .isEqualTo(Lifecycle.State.RESUMED)
-            assertThat(secondEntry?.getLifecycle()?.currentState)
+            assertThat(secondEntry?.lifecycle?.currentState)
                 .isEqualTo(Lifecycle.State.DESTROYED)
         }
     }
@@ -186,9 +186,9 @@ class AnimatedNavHostTest {
         composeTestRule.mainClock.autoAdvance = true
 
         composeTestRule.runOnIdle {
-            assertThat(firstEntry.getLifecycle().currentState)
+            assertThat(firstEntry.lifecycle.currentState)
                 .isEqualTo(Lifecycle.State.CREATED)
-            assertThat(secondEntry.getLifecycle().currentState)
+            assertThat(secondEntry.lifecycle.currentState)
                 .isEqualTo(Lifecycle.State.RESUMED)
         }
     }

--- a/navigation-animation/src/main/java/com/google/accompanist/navigation/animation/AnimatedNavHost.kt
+++ b/navigation-animation/src/main/java/com/google/accompanist/navigation/animation/AnimatedNavHost.kt
@@ -18,7 +18,7 @@ package com.google.accompanist.navigation.animation
 
 import androidx.activity.compose.LocalOnBackPressedDispatcherOwner
 import androidx.compose.animation.AnimatedContent
-import androidx.compose.animation.AnimatedContentScope
+import androidx.compose.animation.AnimatedContentTransitionScope
 import androidx.compose.animation.ContentTransform
 import androidx.compose.animation.EnterTransition
 import androidx.compose.animation.ExitTransition
@@ -78,12 +78,12 @@ public fun AnimatedNavHost(
     modifier: Modifier = Modifier,
     contentAlignment: Alignment = Alignment.Center,
     route: String? = null,
-    enterTransition: (AnimatedContentScope<NavBackStackEntry>.() -> EnterTransition) =
+    enterTransition: (AnimatedContentTransitionScope<NavBackStackEntry>.() -> EnterTransition) =
         { fadeIn(animationSpec = tween(700)) },
-    exitTransition: (AnimatedContentScope<NavBackStackEntry>.() -> ExitTransition) =
+    exitTransition: (AnimatedContentTransitionScope<NavBackStackEntry>.() -> ExitTransition) =
         { fadeOut(animationSpec = tween(700)) },
-    popEnterTransition: (AnimatedContentScope<NavBackStackEntry>.() -> EnterTransition) = enterTransition,
-    popExitTransition: (AnimatedContentScope<NavBackStackEntry>.() -> ExitTransition) = exitTransition,
+    popEnterTransition: (AnimatedContentTransitionScope<NavBackStackEntry>.() -> EnterTransition) = enterTransition,
+    popExitTransition: (AnimatedContentTransitionScope<NavBackStackEntry>.() -> ExitTransition) = exitTransition,
     builder: NavGraphBuilder.() -> Unit
 ) {
     AnimatedNavHost(
@@ -121,12 +121,12 @@ public fun AnimatedNavHost(
     graph: NavGraph,
     modifier: Modifier = Modifier,
     contentAlignment: Alignment = Alignment.Center,
-    enterTransition: (AnimatedContentScope<NavBackStackEntry>.() -> EnterTransition) =
+    enterTransition: (AnimatedContentTransitionScope<NavBackStackEntry>.() -> EnterTransition) =
         { fadeIn(animationSpec = tween(700)) },
-    exitTransition: (AnimatedContentScope<NavBackStackEntry>.() -> ExitTransition) =
+    exitTransition: (AnimatedContentTransitionScope<NavBackStackEntry>.() -> ExitTransition) =
         { fadeOut(animationSpec = tween(700)) },
-    popEnterTransition: (AnimatedContentScope<NavBackStackEntry>.() -> EnterTransition) = enterTransition,
-    popExitTransition: (AnimatedContentScope<NavBackStackEntry>.() -> ExitTransition) = exitTransition,
+    popEnterTransition: (AnimatedContentTransitionScope<NavBackStackEntry>.() -> EnterTransition) = enterTransition,
+    popExitTransition: (AnimatedContentTransitionScope<NavBackStackEntry>.() -> ExitTransition) = exitTransition,
 ) {
 
     val lifecycleOwner = LocalLifecycleOwner.current
@@ -164,7 +164,7 @@ public fun AnimatedNavHost(
     val backStackEntry = visibleEntries.lastOrNull()
 
     if (backStackEntry != null) {
-        val finalEnter: AnimatedContentScope<NavBackStackEntry>.() -> EnterTransition = {
+        val finalEnter: AnimatedContentTransitionScope<NavBackStackEntry>.() -> EnterTransition = {
             val targetDestination = targetState.destination as AnimatedComposeNavigator.Destination
 
             if (composeNavigator.isPop.value) {
@@ -178,7 +178,7 @@ public fun AnimatedNavHost(
             }
         }
 
-        val finalExit: AnimatedContentScope<NavBackStackEntry>.() -> ExitTransition = {
+        val finalExit: AnimatedContentTransitionScope<NavBackStackEntry>.() -> ExitTransition = {
             val initialDestination = initialState.destination as AnimatedComposeNavigator.Destination
 
             if (composeNavigator.isPop.value) {
@@ -243,16 +243,16 @@ public fun AnimatedNavHost(
 @ExperimentalAnimationApi
 internal val enterTransitions =
     mutableMapOf<String?,
-        (AnimatedContentScope<NavBackStackEntry>.() -> EnterTransition?)?>()
+        (AnimatedContentTransitionScope<NavBackStackEntry>.() -> EnterTransition?)?>()
 
 @ExperimentalAnimationApi
 internal val exitTransitions =
-    mutableMapOf<String?, (AnimatedContentScope<NavBackStackEntry>.() -> ExitTransition?)?>()
+    mutableMapOf<String?, (AnimatedContentTransitionScope<NavBackStackEntry>.() -> ExitTransition?)?>()
 
 @ExperimentalAnimationApi
 internal val popEnterTransitions =
-    mutableMapOf<String?, (AnimatedContentScope<NavBackStackEntry>.() -> EnterTransition?)?>()
+    mutableMapOf<String?, (AnimatedContentTransitionScope<NavBackStackEntry>.() -> EnterTransition?)?>()
 
 @ExperimentalAnimationApi
 internal val popExitTransitions =
-    mutableMapOf<String?, (AnimatedContentScope<NavBackStackEntry>.() -> ExitTransition?)?>()
+    mutableMapOf<String?, (AnimatedContentTransitionScope<NavBackStackEntry>.() -> ExitTransition?)?>()

--- a/navigation-animation/src/main/java/com/google/accompanist/navigation/animation/NavGraphBuilder.kt
+++ b/navigation-animation/src/main/java/com/google/accompanist/navigation/animation/NavGraphBuilder.kt
@@ -17,7 +17,7 @@
 package com.google.accompanist.navigation.animation
 
 import android.annotation.SuppressLint
-import androidx.compose.animation.AnimatedContentScope
+import androidx.compose.animation.AnimatedContentTransitionScope
 import androidx.compose.animation.AnimatedVisibilityScope
 import androidx.compose.animation.EnterTransition
 import androidx.compose.animation.ExitTransition
@@ -49,13 +49,13 @@ public fun NavGraphBuilder.composable(
     route: String,
     arguments: List<NamedNavArgument> = emptyList(),
     deepLinks: List<NavDeepLink> = emptyList(),
-    enterTransition: (AnimatedContentScope<NavBackStackEntry>.() -> EnterTransition?)? = null,
-    exitTransition: (AnimatedContentScope<NavBackStackEntry>.() -> ExitTransition?)? = null,
+    enterTransition: (AnimatedContentTransitionScope<NavBackStackEntry>.() -> EnterTransition?)? = null,
+    exitTransition: (AnimatedContentTransitionScope<NavBackStackEntry>.() -> ExitTransition?)? = null,
     popEnterTransition: (
-        AnimatedContentScope<NavBackStackEntry>.() -> EnterTransition?
+        AnimatedContentTransitionScope<NavBackStackEntry>.() -> EnterTransition?
     )? = enterTransition,
     popExitTransition: (
-        AnimatedContentScope<NavBackStackEntry>.() -> ExitTransition?
+        AnimatedContentTransitionScope<NavBackStackEntry>.() -> ExitTransition?
     )? = exitTransition,
     content: @Composable AnimatedVisibilityScope.(NavBackStackEntry) -> Unit
 ) {
@@ -101,13 +101,13 @@ public fun NavGraphBuilder.navigation(
     route: String,
     arguments: List<NamedNavArgument> = emptyList(),
     deepLinks: List<NavDeepLink> = emptyList(),
-    enterTransition: (AnimatedContentScope<NavBackStackEntry>.() -> EnterTransition?)? = null,
-    exitTransition: (AnimatedContentScope<NavBackStackEntry>.() -> ExitTransition?)? = null,
+    enterTransition: (AnimatedContentTransitionScope<NavBackStackEntry>.() -> EnterTransition?)? = null,
+    exitTransition: (AnimatedContentTransitionScope<NavBackStackEntry>.() -> ExitTransition?)? = null,
     popEnterTransition: (
-        AnimatedContentScope<NavBackStackEntry>.() -> EnterTransition?
+        AnimatedContentTransitionScope<NavBackStackEntry>.() -> EnterTransition?
     )? = enterTransition,
     popExitTransition: (
-        AnimatedContentScope<NavBackStackEntry>.() -> ExitTransition?
+        AnimatedContentTransitionScope<NavBackStackEntry>.() -> ExitTransition?
     )? = exitTransition,
     builder: NavGraphBuilder.() -> Unit
 ) {

--- a/navigation-material/src/androidTest/java/com/google/accompanist/navigation.material/BottomSheetNavigatorTest.kt
+++ b/navigation-material/src/androidTest/java/com/google/accompanist/navigation.material/BottomSheetNavigatorTest.kt
@@ -298,7 +298,7 @@ internal class BottomSheetNavigatorTest {
 
         assertThat(navigatorState.transitionsInProgress.value).doesNotContain(backStackEntry1)
         assertThat(navigatorState.transitionsInProgress.value).doesNotContain(backStackEntry2)
-        assertThat(backStackEntry2.getLifecycle().currentState).isEqualTo(Lifecycle.State.RESUMED)
+        assertThat(backStackEntry2.lifecycle.currentState).isEqualTo(Lifecycle.State.RESUMED)
     }
 
     @Test
@@ -387,7 +387,7 @@ internal class BottomSheetNavigatorTest {
         }
 
         composeTestRule.runOnIdle {
-            assertThat(bottomSheetEntry.getLifecycle().currentState).isEqualTo(Lifecycle.State.DESTROYED)
+            assertThat(bottomSheetEntry.lifecycle.currentState).isEqualTo(Lifecycle.State.DESTROYED)
         }
     }
 
@@ -676,7 +676,7 @@ internal class BottomSheetNavigatorTest {
         assertThat(navController.currentBackStackEntry?.destination?.route).isEqualTo(
             sheetDestination
         )
-        assertThat(navController.currentBackStackEntry?.getLifecycle()?.currentState).isEqualTo(Lifecycle.State.RESUMED)
+        assertThat(navController.currentBackStackEntry?.lifecycle?.currentState).isEqualTo(Lifecycle.State.RESUMED)
         assertThat(navigator.navigatorSheetState.isVisible).isTrue()
 
         composeTestRule.onNodeWithTag(sheetLayoutTestTag)

--- a/sample/src/main/java/com/google/accompanist/sample/navigation/animation/AnimatedNavHostSample.kt
+++ b/sample/src/main/java/com/google/accompanist/sample/navigation/animation/AnimatedNavHostSample.kt
@@ -19,7 +19,7 @@ package com.google.accompanist.sample.navigation.animation
 import android.os.Bundle
 import androidx.activity.ComponentActivity
 import androidx.activity.compose.setContent
-import androidx.compose.animation.AnimatedContentScope
+import androidx.compose.animation.AnimatedContentTransitionScope
 import androidx.compose.animation.AnimatedVisibilityScope
 import androidx.compose.animation.ExitTransition
 import androidx.compose.animation.ExperimentalAnimationApi
@@ -100,28 +100,28 @@ fun ExperimentalAnimationNav() {
             enterTransition = {
                 when (initialState.destination.route) {
                     "Red" ->
-                        slideIntoContainer(AnimatedContentScope.SlideDirection.Left, animationSpec = tween(700))
+                        slideIntoContainer(AnimatedContentTransitionScope.SlideDirection.Left, animationSpec = tween(700))
                     else -> null
                 }
             },
             exitTransition = {
                 when (targetState.destination.route) {
                     "Red" ->
-                        slideOutOfContainer(AnimatedContentScope.SlideDirection.Left, animationSpec = tween(700))
+                        slideOutOfContainer(AnimatedContentTransitionScope.SlideDirection.Left, animationSpec = tween(700))
                     else -> null
                 }
             },
             popEnterTransition = {
                 when (initialState.destination.route) {
                     "Red" ->
-                        slideIntoContainer(AnimatedContentScope.SlideDirection.Right, animationSpec = tween(700))
+                        slideIntoContainer(AnimatedContentTransitionScope.SlideDirection.Right, animationSpec = tween(700))
                     else -> null
                 }
             },
             popExitTransition = {
                 when (targetState.destination.route) {
                     "Red" ->
-                        slideOutOfContainer(AnimatedContentScope.SlideDirection.Right, animationSpec = tween(700))
+                        slideOutOfContainer(AnimatedContentTransitionScope.SlideDirection.Right, animationSpec = tween(700))
                     else -> null
                 }
             }
@@ -131,36 +131,36 @@ fun ExperimentalAnimationNav() {
             enterTransition = {
                 when (initialState.destination.route) {
                     "Blue" ->
-                        slideIntoContainer(AnimatedContentScope.SlideDirection.Left, animationSpec = tween(700))
+                        slideIntoContainer(AnimatedContentTransitionScope.SlideDirection.Left, animationSpec = tween(700))
                     "Green" ->
-                        slideIntoContainer(AnimatedContentScope.SlideDirection.Up, animationSpec = tween(700))
+                        slideIntoContainer(AnimatedContentTransitionScope.SlideDirection.Up, animationSpec = tween(700))
                     else -> null
                 }
             },
             exitTransition = {
                 when (targetState.destination.route) {
                     "Blue" ->
-                        slideOutOfContainer(AnimatedContentScope.SlideDirection.Left, animationSpec = tween(700))
+                        slideOutOfContainer(AnimatedContentTransitionScope.SlideDirection.Left, animationSpec = tween(700))
                     "Green" ->
-                        slideOutOfContainer(AnimatedContentScope.SlideDirection.Up, animationSpec = tween(700))
+                        slideOutOfContainer(AnimatedContentTransitionScope.SlideDirection.Up, animationSpec = tween(700))
                     else -> null
                 }
             },
             popEnterTransition = {
                 when (initialState.destination.route) {
                     "Blue" ->
-                        slideIntoContainer(AnimatedContentScope.SlideDirection.Right, animationSpec = tween(700))
+                        slideIntoContainer(AnimatedContentTransitionScope.SlideDirection.Right, animationSpec = tween(700))
                     "Green" ->
-                        slideIntoContainer(AnimatedContentScope.SlideDirection.Down, animationSpec = tween(700))
+                        slideIntoContainer(AnimatedContentTransitionScope.SlideDirection.Down, animationSpec = tween(700))
                     else -> null
                 }
             },
             popExitTransition = {
                 when (targetState.destination.route) {
                     "Blue" ->
-                        slideOutOfContainer(AnimatedContentScope.SlideDirection.Right, animationSpec = tween(700))
+                        slideOutOfContainer(AnimatedContentTransitionScope.SlideDirection.Right, animationSpec = tween(700))
                     "Green" ->
-                        slideOutOfContainer(AnimatedContentScope.SlideDirection.Down, animationSpec = tween(700))
+                        slideOutOfContainer(AnimatedContentTransitionScope.SlideDirection.Down, animationSpec = tween(700))
                     else -> null
                 }
             }
@@ -177,7 +177,7 @@ fun ExperimentalAnimationNav() {
                     when (initialState.destination.route) {
                         "Red" ->
                             slideIntoContainer(
-                                AnimatedContentScope.SlideDirection.Up, animationSpec = tween(700)
+                                AnimatedContentTransitionScope.SlideDirection.Up, animationSpec = tween(700)
                             )
                         else -> null
                     }
@@ -186,7 +186,7 @@ fun ExperimentalAnimationNav() {
                     when (targetState.destination.route) {
                         "Red" ->
                             slideOutOfContainer(
-                                AnimatedContentScope.SlideDirection.Up, animationSpec = tween(700)
+                                AnimatedContentTransitionScope.SlideDirection.Up, animationSpec = tween(700)
                             )
                         else -> null
                     }
@@ -195,7 +195,7 @@ fun ExperimentalAnimationNav() {
                     when (initialState.destination.route) {
                         "Red" ->
                             slideIntoContainer(
-                                AnimatedContentScope.SlideDirection.Down, animationSpec = tween(700)
+                                AnimatedContentTransitionScope.SlideDirection.Down, animationSpec = tween(700)
                             )
                         else -> null
                     }
@@ -204,7 +204,7 @@ fun ExperimentalAnimationNav() {
                     when (targetState.destination.route) {
                         "Red" ->
                             slideOutOfContainer(
-                                AnimatedContentScope.SlideDirection.Down, animationSpec = tween(700)
+                                AnimatedContentTransitionScope.SlideDirection.Down, animationSpec = tween(700)
                             )
                         else -> null
                     }

--- a/sample/src/main/java/com/google/accompanist/sample/systemuicontroller/SystemBarsVisibilitySample.kt
+++ b/sample/src/main/java/com/google/accompanist/sample/systemuicontroller/SystemBarsVisibilitySample.kt
@@ -109,7 +109,7 @@ private fun Sample() {
                             isShowingDropdownMenu = false
                         }
                     ) {
-                        Text("BEHAVIOR_SHOW_BARS_BY_SWIPE")
+                        Text("BEHAVIOR_DEFAULT")
                     }
                     DropdownMenuItem(
                         onClick = {

--- a/sample/src/main/java/com/google/accompanist/sample/systemuicontroller/SystemBarsVisibilitySample.kt
+++ b/sample/src/main/java/com/google/accompanist/sample/systemuicontroller/SystemBarsVisibilitySample.kt
@@ -105,16 +105,7 @@ private fun Sample() {
                     DropdownMenuItem(
                         onClick = {
                             systemUiController.systemBarsBehavior =
-                                WindowInsetsControllerCompat.BEHAVIOR_SHOW_BARS_BY_TOUCH
-                            isShowingDropdownMenu = false
-                        }
-                    ) {
-                        Text("BEHAVIOR_SHOW_BARS_BY_TOUCH")
-                    }
-                    DropdownMenuItem(
-                        onClick = {
-                            systemUiController.systemBarsBehavior =
-                                WindowInsetsControllerCompat.BEHAVIOR_SHOW_BARS_BY_SWIPE
+                                WindowInsetsControllerCompat.BEHAVIOR_DEFAULT
                             isShowingDropdownMenu = false
                         }
                     ) {

--- a/systemuicontroller/src/main/java/com/google/accompanist/systemuicontroller/SystemUiController.kt
+++ b/systemuicontroller/src/main/java/com/google/accompanist/systemuicontroller/SystemUiController.kt
@@ -48,8 +48,8 @@ interface SystemUiController {
     /**
      * Control for the behavior of the system bars. This value should be one of the
      * [WindowInsetsControllerCompat] behavior constants:
-     * [WindowInsetsControllerCompat.BEHAVIOR_SHOW_BARS_BY_TOUCH],
-     * [WindowInsetsControllerCompat.BEHAVIOR_SHOW_BARS_BY_SWIPE] and
+     * [WindowInsetsControllerCompat.BEHAVIOR_SHOW_BARS_BY_TOUCH] (Deprecated),
+     * [WindowInsetsControllerCompat.BEHAVIOR_DEFAULT] and
      * [WindowInsetsControllerCompat.BEHAVIOR_SHOW_TRANSIENT_BARS_BY_SWIPE].
      */
     var systemBarsBehavior: Int

--- a/systemuicontroller/src/sharedTest/kotlin/com/google/accompanist/systemuicontroller/ActivityRememberSystemUiControllerTest.kt
+++ b/systemuicontroller/src/sharedTest/kotlin/com/google/accompanist/systemuicontroller/ActivityRememberSystemUiControllerTest.kt
@@ -246,6 +246,7 @@ class ActivityRememberSystemUiControllerTest {
         }
     }
 
+    @Suppress("DEPRECATION")
     @Test
     @SdkSuppress(minSdkVersion = 30) // TODO: https://issuetracker.google.com/issues/189366125
     fun systemBarsBehavior_showBarsByTouch() {
@@ -275,11 +276,11 @@ class ActivityRememberSystemUiControllerTest {
 
         rule.activityRule.scenario.onActivity {
             systemUiController.systemBarsBehavior =
-                WindowInsetsControllerCompat.BEHAVIOR_SHOW_BARS_BY_SWIPE
+                WindowInsetsControllerCompat.BEHAVIOR_DEFAULT
         }
 
         assertThat(WindowCompat.getInsetsController(window, contentView).systemBarsBehavior)
-            .isEqualTo(WindowInsetsControllerCompat.BEHAVIOR_SHOW_BARS_BY_SWIPE)
+            .isEqualTo(WindowInsetsControllerCompat.BEHAVIOR_DEFAULT)
     }
 
     @Test

--- a/systemuicontroller/src/sharedTest/kotlin/com/google/accompanist/systemuicontroller/ActivitySystemUiControllerTest.kt
+++ b/systemuicontroller/src/sharedTest/kotlin/com/google/accompanist/systemuicontroller/ActivitySystemUiControllerTest.kt
@@ -226,6 +226,7 @@ class ActivitySystemUiControllerTest {
         }
     }
 
+    @Suppress("DEPRECATION")
     @Test
     @SdkSuppress(minSdkVersion = 30) // TODO: https://issuetracker.google.com/issues/189366125
     fun systemBarsBehavior_showBarsByTouch() {
@@ -249,11 +250,11 @@ class ActivitySystemUiControllerTest {
         }
 
         rule.scenario.onActivity {
-            controller.systemBarsBehavior = WindowInsetsControllerCompat.BEHAVIOR_SHOW_BARS_BY_SWIPE
+            controller.systemBarsBehavior = WindowInsetsControllerCompat.BEHAVIOR_DEFAULT
         }
 
         assertThat(WindowCompat.getInsetsController(window, contentView).systemBarsBehavior)
-            .isEqualTo(WindowInsetsControllerCompat.BEHAVIOR_SHOW_BARS_BY_SWIPE)
+            .isEqualTo(WindowInsetsControllerCompat.BEHAVIOR_DEFAULT)
     }
 
     @Test

--- a/systemuicontroller/src/sharedTest/kotlin/com/google/accompanist/systemuicontroller/DialogRememberSystemUiControllerTest.kt
+++ b/systemuicontroller/src/sharedTest/kotlin/com/google/accompanist/systemuicontroller/DialogRememberSystemUiControllerTest.kt
@@ -287,6 +287,7 @@ class DialogRememberSystemUiControllerTest {
         }
     }
 
+    @Suppress("DEPRECATION")
     @Test
     @SdkSuppress(minSdkVersion = 30) // TODO: https://issuetracker.google.com/issues/189366125
     fun systemBarsBehavior_showBarsByTouch() {
@@ -326,11 +327,11 @@ class DialogRememberSystemUiControllerTest {
 
         rule.activityRule.scenario.onActivity {
             systemUiController.systemBarsBehavior =
-                WindowInsetsControllerCompat.BEHAVIOR_SHOW_BARS_BY_SWIPE
+                WindowInsetsControllerCompat.BEHAVIOR_DEFAULT
         }
 
         assertThat(WindowCompat.getInsetsController(window, contentView).systemBarsBehavior)
-            .isEqualTo(WindowInsetsControllerCompat.BEHAVIOR_SHOW_BARS_BY_SWIPE)
+            .isEqualTo(WindowInsetsControllerCompat.BEHAVIOR_DEFAULT)
     }
 
     @Test

--- a/systemuicontroller/src/sharedTest/kotlin/com/google/accompanist/systemuicontroller/DialogSystemUiControllerTest.kt
+++ b/systemuicontroller/src/sharedTest/kotlin/com/google/accompanist/systemuicontroller/DialogSystemUiControllerTest.kt
@@ -245,6 +245,7 @@ class DialogSystemUiControllerTest {
         }
     }
 
+    @Suppress("DEPRECATION")
     @Test
     @SdkSuppress(minSdkVersion = 30) // TODO: https://issuetracker.google.com/issues/189366125
     fun systemBarsBehavior_showBarsByTouch() {
@@ -270,11 +271,11 @@ class DialogSystemUiControllerTest {
 
         rule.scenario.onActivity {
             controller.systemBarsBehavior =
-                WindowInsetsControllerCompat.BEHAVIOR_SHOW_BARS_BY_SWIPE
+                WindowInsetsControllerCompat.BEHAVIOR_DEFAULT
         }
 
         assertThat(WindowCompat.getInsetsController(window, contentView).systemBarsBehavior)
-            .isEqualTo(WindowInsetsControllerCompat.BEHAVIOR_SHOW_BARS_BY_SWIPE)
+            .isEqualTo(WindowInsetsControllerCompat.BEHAVIOR_DEFAULT)
     }
 
     @Test

